tools/flang1/flang1exe/fpp.c

```diff
@@ -376,6 +376,20 @@ skipbl(char *tokval, int flag)
   return toktyp;
 }
 
+extern bool g_list_macros;
+void fpp_display_macros()
```

Comment:
This code looks very similar as in the other file. Personally, I would move it to some header and create `inline` function in there (or a `do..while (0)` preprocessor macro, but that requires some experience). All code duplications are bad and create maintenance nightmare. But since these two files are already plagued with duplication, I'm leaving decision whether to move it or to leave it to your personal taste.